config/config.go

```diff
@@ -223,6 +223,7 @@ type CirconusConfig struct {
 	Broker          string            `toml:"broker"`
 	Hostname        string            `toml:"-"`
 	CheckSearchTags []string          `toml:"check_search_tags"`
+	CheckTags       string            `toml:"check_tags"`
```

Comment:
I know consistency is important and all, so maybe this is the way to go. But, I would recommend a []string type for the Check Tags and the associated TOML to create an array of tag strings. The API internally represents check tags as a slice of strings and this prevents problems for values potentially containing a ",". I know other agents may do this differently so this is just a suggestion.

Reply:
👍 completed in [7690755](https://github.com/circonus-labs/circonus-unified-agent/pull/77/commits/7690755cf367e4b58d90e332f8f601d3d2b51afb)